packages/psss-server/examples.http

```diff
@@ -39,7 +39,7 @@ content-type: application/json
 DELETE http://localhost:8040/v1/weeklyReport/TO?week=2021W07 HTTP/2.0
 
 ### Fetch Confirmed Weekly Report Data
-GET http://localhost:8040/v1/confirmedWeeklyReport?startWeek=2021W07&endWeek=2021W07 HTTP/2.0
+GET http://localhost:8040/v1/confirmedWeeklyReport?startWeek=2021W07&endWeek=2021W07&orgUnitCodes=TO&orgUnitCodes=FJ HTTP/2.0
```

Comment:
We could also pass `orgUnitCodes` as a comma-separated string, which would make the URL easier to read.

On the other hand, this would make invoking the API a tiny bit more complicated since we'd need to stringify an array parameter. I decided to not go this way and keep the programmatic invocation of the API simple, but let me know if you think that it would be preferable